TikTokApi/tiktok.py

```diff
@@ -83,23 +91,23 @@ def clean_up(self):
     def __del__(self):
         try:
             self.browser.clean_up()
-        except:
+        except BaseException:
```

Comment:
As mentioned above, these should be `except Exception`.  Was an autoformatter used to make these changes?  this is functionally equivalent to a bare except, but carries the same risks as a result so it's not actually better.

Reply:
Theese changes come from a merge I think. Anway, I'm going to fix that!